### ISSUE TITLE
Add ability to use locale.time_format

### ISSUE THREAD

**Issue by marksie1988** [custom-cards/custom-card-helpers#35]
The latest version introduces `locale.time_format` please could support be added for this?

**Comment by JonahKr** [comment]
i think i can take care of it in the next weeks or so

**Comment by marksie1988** [comment]
Thanks, It was above my head, I'm still learning typescript but my main focus is python at the moment for work, otherwise I would have spent some time on this.

**Comment by nielsfaber** [comment]
I also like to see this feature added for being able to migrate my project (context: https://github.com/nielsfaber/scheduler-card/issues/322) 👍 

I also don't think the `HomeAssistant.locale` property should be optional.
This breaks a simple call like:
```
computeStateDisplay(hass.localize, hass.states["switch.my_switch"], hass.locale)
```
Due to `hass.locale` being "possibly undefined".

**Comment by marksie1988** [comment]
yes I had the same issue when trying to implement, it was not possible with having it set as possibly undefined.

**Comment by kukudemajia** [comment]
please could add ability to use locale.time_format ?

**Comment by kukudemajia** [comment]
Please could add ability to use locale.time_format ?

**Comment by JonahKr** [comment]
This isn't fully implemented yet @iantrich we should reopen this until all date and time functions got updated

**Comment by iantrich** [comment]
My bad. Thanks for clarifying

**Comment by JonahKr** [comment]
This got implemented in #50 and will be shipped with the next release 👍
